### ISSUE TITLE
How to pass the "click-to-verify-your-email-account" step?

### ISSUE THREAD

**Issue by Sybacom** [leafac/kill-the-newsletter#73]
Hi Leandro!
Firstly thanks so much for your great tool !

But it becomes a big issue that I/we can't verify our email account via clicking on the verification link the newsletter handlers are sending automatically upon subscription.

Any ideas how we can solve that issue?
Cheers

**Comment by zakius** [comment]
That message should arrive in your feed and the link should just work, if that's not the case then there's something wrong with either the message (if the link isn't working as intended) or the mailing service itself (if you aren't receiving the message at all)

if it's the latter you may try using your own instance as there's a chance the public one is being actively blocked by domain by some newsletter senders

**Comment by Sybacom** [comment]
hey, thanks for taking time to reply. 
It worked indeed, 
but there's a small detail here :

you have to subscribe FIRST to the KTN feed you create using your feed reader,
and THEN subscribe to the newsletter with the relevant KTN email account.

Maybe @leafac it would help to set clear that order of acting?
I didn't think of it before !

**Comment by zakius** [comment]
From my experience and understanding of the code this isn't the case and to verify that I just created new box, sent message to it and then subscribed to the feed, the message was added to the feed and displayed properly so you can do it in either order

**Comment by Sybacom** [comment]
hmm, that's curious, I did so many times the last (1st subscribe with KTN email and then add the feed to my feedly),  and I didn't receive the automatic "click to verify"  email in my feed. So couldn't subscribe to the newsletters.

And now it worked like a charm! 
So sorry bothering you guys.  Maybe something changed recently? Like Feedly maybe changed sth? 
It would be too harsh to accept that... I did sth wrong! I mean it's only just 2 steps after all
